psi4/driver/procrouting/proc.py

```diff
@@ -5086,7 +5086,7 @@ def run_cepa(name, **kwargs):
     if ref_wfn is None:
         ref_wfn = scf_helper(name, **kwargs)  # C1 certified
 
-    if core.get_option('FNOCC', 'USE_DF_INTS') == False:
+    if core.get_option('FNOCC', 'USE_DF_INTS') is False:
```

Comment:
```suggestion
    if core.get_option('FNOCC', 'USE_DF_INTS') == 0:
```

Reply:
You fixed one of these important lines, but the other one needs to be fixed, too.